### PR TITLE
Complete example Autocrypt Setup Message in appendix

### PR DESCRIPTION
**Pull request by juga0**

### CODE REVIEW

File: doc/level1.rst [hunk 1]
```diff
@@ -934,46 +934,243 @@ Example Setup Message
...
-   Content-Type: application/autocrypt-key-backup
-   Content-Disposition: attachment
+    To set up your new device for Autocrypt, please follow the instuctions that should be presented by your new device.
 
-   Possibly a descriptive text, informing users about this file's
-   contents.
-   -----BEGIN PGP MESSAGE-----
-   Passphrase-Format: numeric9x4
-   Passphrase-Begin: 31
+    You can keep this message and use it as a backup for your secret key. If you want to do this, you should write down the password and store it securely.
```
Comment (dkg) [inline doc/level1.rst]: Why are we using "password" here?  shouldn't this be "setup code" ?

File: doc/level1.rst [hunk 2]
```diff
@@ -934,46 +934,243 @@ Example Setup Message
...
+    --Y6fyGi9SoGeH8WwRaEdC6bbBcYOedDzrQ
+    Content-Type: application/autocrypt-setup
+    Content-Disposition: attachment; filename="autocrypt-key-backup.html"
```
Comment (dkg) [inline doc/level1.rst]: why is filename="autocrypt-key-backup.html" ?  shouldn't this be "autocrypt-setup-message.html" for consistency in interface?

Comment (Valodim) [reply doc/level1.rst]: I would omit the html trick from this example. It's not obvious how this works or why it is useful, and in the spec we only mention it in reference to a thought in an external document.

**Comment by juga0** [comment]
Last commit fixes @dkg comments

**Pull request merged**